packages/local_auth/local_auth_android/CHANGELOG.md

```diff
@@ -1,8 +1,14 @@
-## 1.0.1
+## 1.0.2
 
-* Fixes `getEnrolledBiometrics` to match documented behaviour: 
+* Fixes `getEnrolledBiometrics` to match documented behaviour:
   Present biometrics that are not enrolled are no longer returned.
-* `getEnrolledBiometrics` now only returns `weak` and `strong` biometric types.   
+* `getEnrolledBiometrics` now only returns `weak` and `strong` biometric types.
+* `deviceSupportsBiometrics` now returns the correct value regardless of enrollment state.
+* Adopts `Object.hash`.
```

Comment:
This line got duplicated from 1.0.1 in the merge resolution, and should be deleted.